executor/src/witgen/jit/function_cache.rs

```diff
@@ -103,13 +103,26 @@ impl<'a, T: FieldElement> FunctionCache<'a, T> {
 
                 log::trace!("Compiling effects...");
 
-                compile_effects(
-                    self.column_layout.first_column_id,
-                    self.column_layout.column_count,
-                    &known_inputs,
-                    &code,
-                )
-                .unwrap()
+                let compiled_jit =
+                    !matches!(std::env::var("POWDR_JIT_INTERPRETER"), Ok(val) if val == "1");
```

Comment:
I don't particularly like the use of environment variables. Could we move them as far up as possible? In my opinion, it should just be a configuration setting in the constructor of FuncitonCache.

Reply:
i removed the other env var (for fully disabling jit), can we just keep this one as is for the moment (while interpreter is fully integrated) for easy testing?

Reply:
To be honest, I would prefer to remove the interpreter from function_cache for the time being...

Reply:
ok, removed it completely